test/test_prototype_builtin_datasets.py

```diff
@@ -143,3 +144,19 @@ def test_extra_label(self, dataset_mock, config):
             ("unused", bool),
         ):
             assert key in sample and isinstance(sample[key], type)
+
+
+@parametrize_dataset_mocks(DATASET_MOCKS["gtsrb"])
+class TestGTSRB:
+    def test_label_matches_path(self, dataset_mock, config):
+        # We read the labels from the csv files instead. But for the trainset, the labels are also part of the path.
+        # This test makes sure that they're both the same
+        if config["split"] != "train":
+            return
```

Comment:
@pmeier that test is only relevant for the training data. is there a way to only call this test with a specific config?

Reply:
Not yet unfortunately. I would say let's keep it they way you have right now and revisit this later when we extend the test suite in general. Still, mini nit: `config` is a bunch

```suggestion
        if config.split != "train":
            return
```